kubeflow/trainer/rhai/transformers.py

```
@@ -391,16 +391,95 @@ def __init__(self, cloud_remote_storage_uri: Optional[str] = None) -> None:
+            class ProgressCallback(Callback):
+                def __init__(self, name, total_size):
+                    super().__init__()
+                    self.name = name
+                    self.total_size = total_size
+                    self.last = time.time()
+                    self.value = 0
+                    print(
+                        f"[Kubeflow] {name} size: {total_size / (1024 * 1024):.1f} MB", flush=True
+                    )
+
+                def branched(self, path_1, path_2, **kwargs):
```

Ruff (0.14.14)

[warning] Unused method argument: `path_1`

(ARG002)